Makefile

```diff
@@ -203,7 +203,7 @@ ifndef PULL_SECRET_PATH
 export PULL_SECRET_PATH = $(HOME)/Downloads/crc-pull-secret
 endif
 ifndef BUNDLE_PATH
-export BUNDLE_PATH = $(HOME)/Downloads/crc_libvirt_$(OPENSHIFT_VERSION).$(BUNDLE_EXTENSION)
+export BUNDLE_PATH = $(HOME)/Downloads/crc_libvirt_$(OPENSHIFT_VERSION)_amd64.$(BUNDLE_EXTENSION)
```

Comment:
@jsliacan This should be part of different commit since here you are adding the arch explicit.

Reply:
@praveenkumar Thanks for noticing. Should I remove it or create an `ARCH` var in expectation of arm or keep it hardcoded like it is?

Reply:
@jsliacan better to use `GOARCH` env from the Makefile.